pkg/utils/baremetal/bootstrap.go

```diff
@@ -12,11 +15,13 @@ import (
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/rest"
 	clientwatch "k8s.io/client-go/tools/watch"
+
+	"github.com/openshift/installer/pkg/infrastructure/baremetal"
 )
 
 // WaitForBaremetalBootstrapControlPlane will watch baremetalhost resources on the bootstrap
 // and wait for the control plane to finish provisioning.
-func WaitForBaremetalBootstrapControlPlane(ctx context.Context, config *rest.Config) error {
+func WaitForBaremetalBootstrapControlPlane(ctx context.Context, config *rest.Config, dir string) error {
```

Comment:
Need to pass the extra arg in `cmd/openshift-install/create.go`. I thought I saw that last time I reviewed; it must have dropped out in a rebase.